pages/form/WorkArrangement.jsx

```diff
@@ -0,0 +1,186 @@
+    >
+      <Box 
+        pos="absolute" 
+        top={["2", "5"]} 
+        left={["2", "10"]}
+      >
+        <BackButton
+          href="/"
+          onClick={() => {
+            saveDataAndShowLog("Back button clicked");
+          }}
+        />
+      </Box>
+      <Q3Cloud />
+      <Heading 
+        mt={10} 
+        mb={10}
+      >
+      What is your usual work arrangement?
+      </Heading>
+      <Box 
+        justify="center" 
+        borderWidth="2px" 
+        borderRadius="lg" 
+        minW="750px" 
+        py="8%"
+      >
+        <Flex 
+          px="20"
+          direction={"column"}
+        >
+          <Text 
+            fontWeight="500" 
+            fontSize="18px"
+          >
+            Select your work arrangement
+          </Text>
+          <RadioGroup 
+            mt={12} 
+            w="100%" 
+            textAlign="left" 
+            onChange={e => setWorkMode(e)} value={workMode}
+          >
+            <Radio 
+              mb={5} 
+              name={workFromHome} 
+              id={workFromHome} 
+              value={workFromHome}
+            >
+              <Text 
+                fontSize={[18, 20]} 
+                fontWeight={700}
+              >
+                {workFromHome}
+              </Text>
+            </Radio>
+
+            <Collapse in={workMode === workFromHome}>
+              <Text 
+                fontSize={[15, 17]} 
+                px="20px" 
+                py="12px" 
+                bg="#D4EDDA"
+                borderRadius="lg" 
+                textAlign="center"
+              >
+                We will use this information to calculate the emissions you save by working at home.
+              </Text>
+            </Collapse>
+
+            <br/>
+
+            <Radio 
+              mt={5} 
+              mb={5} 
+              name={workOnSite} 
+              id={workOnSite} 
+              value={workOnSite}
+            >
+              <Text 
+                fontSize={[18, 20]} 
+                fontWeight={700}
+              >
+              {workOnSite}
+              </Text>
+            </Radio>
+
+            <br/>
+            
+            <Radio 
+              mt={5} 
+              name={hybridMode} 
+              id={hybridMode} 
+              value={hybridMode}
+            >
+              <Text 
+                fontSize={[18, 20]} 
+                fontWeight={700}
+              >
+                {hybridMode}
+              </Text>
+            </Radio>
+          </RadioGroup>
+
+          <Flex 
+            justify={"end"} 
+            mt={10}
+          >
+            <LinkButton
+              disabled={!workMode}
+              href={"/form/DaysOfTheWeekSelection"}
+              width={"105px"}
+              topMargin="0"
+              H="55px"
+              justifySelf="right"
+              onClick={() => saveDataAndShowLog("Next button clicked")}
+            >
+              Save
```

Comment:
I think we've reverted back to using 'Next' for `LinkButton` text;
see [figma comment here](https://www.figma.com/file/Y42AKuHDW46LiLDaTOs6sM?node-id=978:55230#162988506) 
```suggestion
              Next
```